tools/update-pdfjs

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
```

Comment:
It's always a good idea to make sure you catch unexpected errors and don't plough on through the script by adding (at least) `set -eu` at the top.